ui/components/EthersInput.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export default function EthersInput({ onChange, ...props}: any) {
+    const onInputChange = (e: any) => {
+        const value = e.target.value
+        const isValid = value.toString().match(/^(\d*\.{0,1}\d{0,18}$)/)
```

Comment:
I really like this, is everyone else comfortable with regular expressions? As Aahna is adding Jest, this is the kind of thing that should have a unit test wrapped around it (with a lot of cases), but it works and it is neat 👍

Reply:
@Caranell Now that #96 has been merged, could you add a few unit tests of this function.  That way we can see some human-readable examples valid/invalid values 😄

Reply:
Sure! Added tests